sib200/_default_template_yaml

```diff
@@ -0,0 +1,21 @@
+group: sib200
+dataset_path: Davlan/sib200
+fewshot_config:
+  sampler: first_n
+output_type: multiple_choice
+should_decontaminate: true
+doc_to_decontamination_query: "{{text}}"
+doc_to_text: "Topic Classification: science/technology, travel, politics, sports, health, entertainment, geography.\n\nThe label of [{{text.strip()}}] is" 
```

Comment:
Where is this prompt from? Other multiple_choice / text classification modules in the harness use a `f"{text} Question: ... Answer:"` format (e.g. MNLI, MMLU). The original paper also uses a different format `"Is this a piece of news regarding {{“science, technology, travel, politics, sports, health, entertainment, or geography”}}? {{INPUT}}"`.

Reply:
We used prompting from this paper 
https://arxiv.org/pdf/2401.13303.pdf
<img width="762" alt="image" src="https://github.com/sambanova/lm-evaluation-harness/assets/137233191/a2219e6e-d4c7-46a4-862c-5c759e48680e">
<img width="669" alt="image" src="https://github.com/sambanova/lm-evaluation-harness/assets/137233191/c75e4645-4768-4655-bc03-2fb5caa46c15">

Reply:
I am down to change if if you all think!